rasa_nlu/data_router.py

```diff
@@ -34,7 +33,15 @@ def __init__(self, config, component_builder):
         self.token = config['token']
         self.emulator = self.__create_emulator()
         self.component_builder = component_builder if component_builder else ComponentBuilder(use_cache=True)
-        self.model_store = self.__create_model_store()
+        self.agent_store = self.__create_agent_store()
+
+    @staticmethod
+    def _latest_agent_model(agent_path):
+        """Retrieves the latest trained model for an agent"""
+        agent_models = {model[6:]: model for model in os.listdir(agent_path)}
```

Comment:
need to remove the magic number (e.g. define "model_" as a prefix constant somewhere and take its length here).